docs/release/3.4/release-3-4-34.md

```diff
@@ -0,0 +1,218 @@
+OSG Software Release 3.4.34
+===========================
+
+**Release Date**: 2019-08-29    
+**Supported OS Versions:** EL7, EL6
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   XCache 1.1.1: This release includes packages for ATLAS and CMS XCaches as well as Stash Origin HTTP/S support.
+-   osg-configure 2.4.1: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid
```

Comment:
I'd still like to see the formatting change of `OSG Configure` to match everything else

```suggestion
-   OSG Configure 2.4.1: improvements for Slurm, PBS Pro, BOSCO, Frontier Squid
```

Reply:
Resolve offline